trunk/SUAVE/Attributes/Solids/Acrylic.py

```diff
@@ -0,0 +1,29 @@
+# Acrylic.py
```

Comment:
File needs doxygen tags and documentation.

Reply:
Added.